packages/SwingSet/jsconfig.json

```diff
@@ -10,7 +10,7 @@
     "declaration": true,
     "emitDeclarationOnly": true,
 */
-    "downlevelIteration": true,
+/*    "downlevelIteration": true,*/
```

Comment:
If we're going to touch this, we might as well delete it.

 1. it controls translation, which we don't do; we only use tsc for checking
 2. it seems to be a project level option; it doesn't seem to belong inside compilerOptions

ref https://www.typescriptlang.org/tsconfig#downlevelIteration

Reply:
fixed in new commit

Reply:
> If we're going to touch this, we might as well delete it.

There was some reason I needed this option (there were `tsc` validation errors without it).  I'm fine with removing it, but be aware that if `for...of` shows a Typescript error, it may tell you to reenable it.

Reply:
See: https://github.com/endojs/endo/pull/690#discussion_r620526712